docs/src/components/ReactExample/components/Board.tsx

```diff
@@ -96,6 +104,18 @@ const Board = ({
               Playground
             </ButtonLink>
           )}
+          {variants.length > 0 && (
+            <Popover content={<Variants exampleVariants={variants} onChange={onChangeVariant} />}>
+              <ButtonLink
+                type="secondary"
+                asComponent="button"
```

Comment:
For me the scroll position still changes on click, not sure why because now it's just a button.

Reply:
weird, that it does not happen locally

Reply:
But it happens for you on staging, right? For me it happens locally as well, maybe you need to rebuild components. 🤷  Something in Popover is causing this, not sure what, because I'm getting the same behavior for dark mode button, and regardless of whether the component is ButtonLink, Button, regular `button` or anything else.

It doesn't throw my scrolling off all the way, just part of the way, and the amount seems inconsistent. There's also a very uncomfortable delay between clicking and actually opening the popover, maybe this is related, and it's something that was always bothering me about Popover.

Also, probably unrelated, but see if you can do something about these Poppper warnings, I created a [ticket](https://jira.kiwi.com/browse/ORBIT-1918) for this.

<img width="448" alt="Screen Shot 2021-09-26 at 14 52 09" src="https://user-images.githubusercontent.com/471278/134809054-c960f8a0-2354-4bf7-846e-863c27cfb174.png">

Reply:
yep, started to have that after the rebuild. Looks like refactor to Popper caused that